.github/workflows/tests.yml

```diff
@@ -1,6 +1,12 @@
 name: tests
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '!*.*.*'
+  pull_request:
```

Comment:
The pull request list is empty. I would set it to `pull_request: ~` to avoid problems.

Reply:
Updated it to 
```
  pull_request:
    branches:
      - '**'
```

WDYT?